### ISSUE TITLE
Not showing AsciiDoc format README.adoc

### ISSUE THREAD

**Issue by venkatperi** [npm/registry-issue-archive#191]
Package https://www.npmjs.com/package/arrayp uses a AsciiDoc formatted README.adoc. Shows up on github, and in the installed package, not on the npm website.

**Comment by Haroenv** [comment]
I couldn’t find the reference, but I remember seeing somewhere that the README can only be markdown for npm. 

The registry _does_ contain the README data and its filename though: https://registry.npmjs.com/arrayp

**Comment by soldair** [comment]
Yeah. We support markdown at this time. I'm not sure if there is a task to add support for more readme formats but we'll discuss it internally.

is there a nice module for converting asciidoc to markdown? all of the asciidoc modules on npm have no readmes lol
@bcoe not sure what you think
